mmv1/products/vpcaccess/Connector.yaml

```diff
@@ -46,11 +46,17 @@ examples:
     primary_resource_id: 'connector'
     vars:
       name: 'vpc-con'
+      network_name: 'default'
+    test_vars_overrides:
+      network_name: 'acctest.BootstrapSharedServiceNetworkingConnection(t, "vpc-access-connector")'
```

Comment:
Or, I could name this `"vpc-access"` and if any other vpcaccess acceptance tests need to use a special VPC they should use the same, service-wide bootstrapped VPC? I'm not sure what the intended usage of BootstrapSharedServiceNetworkingConnection is.

Reply:
I think what you have is probably fine. Unless things have changed significantly, anything using bootstrapped networks generally needs to use a per-test id.